contrib/blueflood-docker/configs/ES-Setup/index_settings.json

```diff
@@ -0,0 +1,27 @@
+{
```

Comment:
If the goal to is to keep this independent, I guess it makes sense, but I still feel if these files change, someone has to remember to change them here.